docs/writing-a-test.md

```diff
@@ -22,60 +55,84 @@ Let's say you have an activity layout that represents a welcome screen:
 </LinearLayout>
 ```
 
-We want to write a test that asserts that when a user clicks on a button, the app launches the LoginActivity.
+We want to write a test that asserts that when the user clicks on the "Login" button, the app launches the `LoginActivity`.
+
+To achieve this, we can check that the correct [`Intent`](https://developer.android.com/reference/android/content/Intent) is started when the click is performed. Since Robolectric is a unit testing framework, the `LoginActivity` will not actually be started.
```

Comment:
Sure. For now, when updating documentation, I tried to keep the same structure as the current file.
But it would definitely be useful to have a common format for every markdown file.
I'll submit a dedicated PR for that.